lib/dom-parser.js

```diff
@@ -17,22 +17,16 @@ var ParseError = conventions.ParseError;
 var XMLReader = sax.XMLReader;
 
 /**
- * Normalizes line ending according to <https://www.w3.org/TR/xml11/#sec-line-ends>:
+ * Normalizes line ending according to [https://www.w3.org/TR/xml11/#sec-line-ends](https://www.w3.org/TR/xml11/#sec-line-ends):
+ * XML parsed entities are often stored in computer files which, for editing convenience, are organized into lines. These lines
+ * Are typically separated by some combination of the characters CARRIAGE RETURN (#xD) and LINE FEED (#xA). To simplify the tasks
+ * of applications, the XML processor must behave as if it normalized all line breaks in external parsed Entities (including the
+ * document entity) on input, before parsing, by translating all of the following to a single #xA Character:
  *
- * > XML parsed entities are often stored in computer files which,
- * > for editing convenience, are organized into lines.
- * > These lines are typically separated by some combination
- * > of the characters CARRIAGE RETURN (#xD) and LINE FEED (#xA).
- * >
- * > To simplify the tasks of applications, the XML processor must behave
- * > as if it normalized all line breaks in external parsed entities (including the document entity)
- * > on input, before parsing, by translating all of the following to a single #xA character:
- * >
- * > 1. the two-character sequence #xD #xA
- * > 2. the two-character sequence #xD #x85
- * > 3. the single character #x85
- * > 4. the single character #x2028
- * > 5. any #xD character that is not immediately followed by #xA or #x85.
```

Comment:
Ouch, why was the block quote dropped from the docstring?

If this is what the plugin does, I think we need to add `@prettierignore` to this doc comment.

Reply:
Yeah, I'd use `@prettierignore` for this; there's a limit of how much magic can the plugin handle and, although using the `> ` as a prefix and running the same logic as for descriptions wouldn't be complicated, I don't want to open the door to "supports markdown", because fully supporting it would be a pain :P.

Reply:
Also byproduct of the other plugin.